### PR TITLE
Make sure the VALUES clause in a subselect does not overwrite the type.

### PR DESCRIPTION
**Pull request by joachimvh**
Added corresponding test files.
Also fixed author field in package.json to not be an array.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,9 +2,7 @@
   "name": "sparqljs",
   "version": "1.5.1",
   "description": "A parser for the SPARQL query language",
-  "author": [
-    "Ruben Verborgh <ruben.verborgh@gmail.com>"
-  ],
+  "author": "Ruben Verborgh <ruben.verborgh@gmail.com>",
```
Comment (RubenVerborgh) [inline package.json]: Can you revert this? (no need to change)

File: queries/sub_values.sparql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+PREFIX : <http://example.org/>
+SELECT ?s ?o {
+{
+    SELECT * WHERE {
+        ?s ?p ?o .
+    }
+    VALUES (?o) { (:b) }
+}
+}
```
Comment (RubenVerborgh) [inline queries/sub_values.sparql]: Can you add a newline?

File: test/parsedQueries/sub_values.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+          "?o": "http://example.org/b"
+        }
+      ]
+    }
+  ],
+  "type": "query",
+  "prefixes": {
+    "": "http://example.org/"
+  }
+}
```
Comment (RubenVerborgh) [inline test/parsedQueries/sub_values.json]: Can you add a newline?

**Comment by joachimvh** [comment]
Fixed

**Comment by RubenVerborgh** [comment]
Thanks, merged as 5a2d6283c3b5dc3014103d026b548977bae8ac4a.

**Comment by RubenVerborgh** [comment]
Released as https://github.com/RubenVerborgh/SPARQL.js/releases/tag/v1.5.2

**Pull request closed without merging**